pkg/models/html.go

```diff
@@ -0,0 +1,78 @@
+package models
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"html/template"
+	"io"
+
+	"github.com/cmsgov/easi-app/pkg/sanitization"
+)
+
+// HTML represents html code. It is sanitized to only allow specific tags
+type HTML string
+
+// ToTemplate converts the HTML type to a template.HTML struct
+func (html HTML) ToTemplate() template.HTML {
+	// Note, we sanitize when the data is Unmarshaled. However, if desired it could be sanitized at this entrypoint as well
+	return template.HTML(html) //nolint //the html is sanitized when it is Unmarshaled
+}
+
+// ValueOrEmptyString returns either the value of the html or an empty string if nil
+func (html *HTML) ValueOrEmptyString() string {
+	if html != nil {
+		return string(*html)
+	}
+	return ""
+}
+
+// ValueOrEmptyHTML returns either the value of the html or an empty HTML type if nil
+func (html *HTML) ValueOrEmptyHTML() HTML {
+	if html != nil {
+		return *html
+	}
+	return ""
+}
+
+// StringPointer casts an HTML pointer to a string pointer
+func (html *HTML) StringPointer() *string {
+	return (*string)(html)
+}
+
+// UnmarshalGQLContext unmarshals the data from graphql to the HTML type
+func (html *HTML) UnmarshalGQLContext(ctx context.Context, v interface{}) error {
```

Comment:
It doesn't look like you need the context - could you just use [`UnmarshalGQL`](https://pkg.go.dev/github.com/99designs/gqlgen/graphql#Unmarshaler)?

Reply:
It would be valid to use `UnmarshalHTML`, but  I decided to keep this method signature if for nothing more than an example of how to use the value that also passes context. It doesn't seem to have an additional impact to use context on the GQL side, so I left this in case we want to use context in the future.